charts/posthog/templates/plugins-deployment.yaml

```diff
@@ -89,16 +89,16 @@ spec:
           value: {{ template "posthog.pgbouncer.url" . }}
         - name: USING_PGBOUNCER
           value: 'true'
+        - name: POSTHOG_REDIS_HOST
+          value: {{ template "posthog.redis.host" . }}
+        - name: POSTHOG_REDIS_PORT
+          value: {{ include "posthog.redis.port" . | quote }}
```

Comment:
Why do we need the `| quote` here  while for the host we don't?

Reply:
I just moved it, no idea what should it be?